packages/insomnia-app/app/ui/redux/modules/global.tsx

```diff
@@ -599,6 +603,9 @@ export const exportWorkspacesToFile = (workspaceId: string | undefined = undefin
               'json',
             );
             break;
+
+          default:
+            unreachableCase(`selected export format "${selectedFormat}" is invalid`);
```

Comment:
```suggestion
            unreachableCase(selectedFormat, `selected export format "${selectedFormat}" is invalid`);
```

we need it here too @develohpanda

Reply:
oh!